_posts/2020-10-12-sparklyr-flint-0.2.0-released/sparklyr-flint-0.2.0.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "sparklyr.flint 0.2: ASOF Joins, OLS Regression, and Additional Summarizers"
+description: |
+  We are excited to announce a number of powerful, new functionalities and improvements which are now part of sparklyr.flint 0.2!
+author:
+  - name: Yitao Li
+    url: https://github.com/yitao-li
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com
+slug: sparklyr.flint-0.2.0
+date: 10-12-2020
+categories:
+  - R
+  - Packages/Releases
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/sparklyr-flint-0.2.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+Since [`sparklyr.flint`](https://cran.r-project.org/web/packages/sparklyr.flint/index.html), a [`sparklyr`](https://sparklyr.ai) extension for leveraging [Flint](https://github.com/twosigma/flint) time series functionalities through `sparklyr`, was [introduced](https://blogs.rstudio.com/ai/posts/2020-09-07-sparklyr-flint) in September, we have made a number of enhancements to it, and have successfully submitted `sparklyr.flint` 0.2 to CRAN recently.
+
+In this blog post, we highlight the following new features and improvements from `sparklyr.flint` 0.2:
+
+ * [ASOF Joins](#asof-joins) of Timeseries RDDs
+ * [OLS Regression](#ols-regression)
+ * [Additional Summarizers](#additional-summarizers)
+ * [Better Integration With `sparklyr`](#better-integration-with-sparklyr)
+
+## ASOF Joins
+
+For those unfamiliar with the term, ASOF joins are temporal join operations based on inexact matching of timestamps. Within the context of [Apache Spark](https://spark.apache.org), a join operation, loosely speaking, matches records from two data frames (let's call them `left` and `right`) based on some criteria. A temporal join implies matching records in `left` and `right` based on timestamps, and with inexact matching of timestamps permitted, it is typically useful to join `left` and `right` along one of the following temporal directions:
```

Comment:
interesting ... to me, `ASOF` sounds more like you have timestamps with differing semantics, as in data warehouses (actual time vs valid time, I think, this might have been) ... but what you describe sounds different ...

Still wondering if there might be a term that fits better than "inexact matching" ... it's more like some "boundary-based" matching, isn't it (if that term existed)?  Is there a math term that would fit this behavior...? ;-)